packages/strymonas-pure/strymonas-pure.2.1.1/opam

```diff
@@ -0,0 +1,32 @@
+
+opam-version: "2.0"
+synopsis: "Stream fusion, to completeness"
+maintainer: "tomoaki.kobayashi.t3@alumni.tohoku.ac.jp"
+authors: [
+  "Oleg Kiselyov"
+  "Tomoaki Kobayashi"
+  "Aggelos Biboudis"
+  "Nick Palladinos"
+  "Yannis Smaragdakis"
+]
+license: "MIT"
+homepage: "https://strymonas.github.io/"
+bug-reports: "https://github.com/strymonas/strymonas-ocaml/issues"
+dev-repo: "git+https://github.com/strymonas/strymonas-ocaml.git"
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "ocamlfind" {build}
+]
+build: [
+  [make "pure"]
```

Comment:
I think this needs to be run from inside the `lib` folder, or the main makefile needs to be updated

Reply:
```suggestion
  [make "-C" "lib/" "pure"]
```

this maybe??